app/javascript/react/components/CoffeeshopsShowContainer.js

```diff
@@ -25,14 +28,37 @@ const CoffeeshopsShowContainer = (props) => {
     }
   }
 
+  const fetchCoffeeshopReviews = async () => {
+    try {
+      const response = await fetch(`/api/v1/coffeeshops/${props.match.params.id}/reviews`)
+      if(!response.ok) {
+        const errorMessage = `${response.status}: ${response.statusText}`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const body = await response.json()
+      setReviews(body.reviews)
+    } catch (err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
+  }
+
   useEffect( () => {
     fetchCoffeeshop()
+    fetchCoffeeshopReviews()
   }, [])
 
+  const reviewsComponents = !reviews ? [] : reviews.map( (review) => {
+    return (
+      <ReviewTile key={review.id} review={review} />
+    )
+  })
```

Comment:
this is where the ternary operator starts to lose value, but its still fine. normally though we have a heeuristic of keeping ternaries to one line as often as possible. I think this is straightforward to follow, so I'll take it!